packages/next/src/server/route-modules/app-route/module.ts

```diff
@@ -369,11 +368,14 @@ export class AppRouteRouteModule extends RouteModule<
               handlerContext
             )
           } catch (err) {
-            if (isPrerenderInterruptedError(err)) {
+            if (prospectiveController.signal.aborted) {
               // the route handler called an API which is always dynamic
               // there is no need to try again
               prospectiveRenderIsDynamic = true
-            } else if (process.env.NEXT_DEBUG_BUILD) {
+            } else if (
+              process.env.NEXT_DEBUG_BUILD ||
+              process.env.__NEXT_VERBOSE_LOGGING
```

Comment:
Do we need a separate flag for this?

Reply:
debug build already exists. I added verbose logging a little while back so we can have a flag that is named in a way that makes sense for it having some effect in dev

Reply:
ah i see, I didn't realize we already had this flag